src/lambda/commands/deploySamApplication.ts

```diff
@@ -49,9 +62,13 @@ export async function deploySamApplication({
                     baseDir: undefined,
                     templatePath: template.fsPath,
                     invoker
-                })
-            // TODO: Add nls support
-            restParams.outputChannel.appendLine('Building SAM Application...')
+                }
+            )
+
+            restParams.outputChannel.appendLine(localize(
```

Comment:
Should we consider moving these calls to a channelLogger?

Reply:
Seems reasonable, but not necessary as part of this PR